services/storage-system/pkg/config/config.go

```diff
@@ -73,6 +73,7 @@ type Drivers struct {
 }
 
 type OCISDriver struct {
+	MetadataBackend string `yaml:"metadata_backend" env:"STORAGE_SYSTEM_OCIS_METADATA_BACKEND" desc:"Backend to use for metadata. Can be 'xattrs' or 'ini'."`
```

Comment:
This will appear in the docs once merged. (It's automated :) )  We should aim for "understandable" options. Maybe something like this?

```suggestion
	MetadataBackend string `yaml:"metadata_backend" env:"STORAGE_SYSTEM_OCIS_METADATA_BACKEND" desc:"Backend to use for metadata. Can be 'xattrs' (use extended attributes to store file metadata) or 'ini' (use a dedicated file to store file metadata). Defaults to 'xattrs'. "`
```

We can add more documentation later when the feature is more mature.
cc @mmattel

Reply:
I allowed myself to adapt the text a bit.

Where I am not sure atm, but would like to note, if we should add as last sentence like:

`ini` is currently not active and will be available in the future.

This would prevent readers to try something that is not available.